compiler/src/iree/compiler/Codegen/Common/TransformDialectJitterPass.cpp

```diff
@@ -153,7 +154,7 @@ static Value tileAndFuseAndDistributeImpl(
 
 template <typename TilingTransformOp = TileToForeachThreadOp,
           typename BatchOrIterativeFusion = BatchFusionSpec>
-static Value tfdWithTileSizes(
+static Value buildTFDWithTileSizes(
```

Comment:
Could you please add some comments to these functions. THe name is not clear and I have no idea what these are doing.